tornettools/generate.py

```diff
@@ -262,7 +315,7 @@ def __tgen_client(args, network, name, country, tgenrc_fname):
 
     process = {}
     process["path"] = "{}/bin/tor".format(SHADOW_INSTALL_PREFIX)
-    process["args"] = __format_tor_args(name)
+    process["args"] = __format_tor_args(None)
```

Comment:
Can we add a comment about why we pass `None` here?

Reply:
It's set to `None` since clients don't need a nickname, and the hostname is too long to be a valid tor nickname. I'll add a comment here.